deploy/crds/wildfly.org_wildflyservers_crd.yaml

```diff
@@ -24,486 +24,1039 @@ spec:
+                    the WildFlyServer application
                   properties:
-                    medium:
-                      description: 'What type of storage medium should back this directory.
-                        The default is "" which means to use the node''s default medium.
-                        Must be an empty string (default) or Memory. More info: https://kubernetes.io/docs/concepts/storage/volumes#emptydir'
+                    name:
+                      type: string
+                    podIP:
                       type: string
-                    sizeLimit:
-                      description: 'Total amount of local storage required for this
-                        EmptyDir volume. The size limit is also applicable for memory
-                        medium. The maximum usage on memory medium EmptyDir would
-                        be the minimum value between the SizeLimit specified here
-                        and the sum of memory limits of all containers in a pod. The
-                        default is nil which means that the limit is undefined. More
-                        info: http://kubernetes.io/docs/user-guide/volumes#emptydir'
+                    state:
+                      description: Represent the state of the Pod, it is used especially
+                        during scale down.
+                      enum:
+                      - ACTIVE
+                      - SCALING_DOWN_RECOVERY_INVESTIGATION
+                      - SCALING_DOWN_RECOVERY_DIRTY
+                      - SCALING_DOWN_CLEAN
                       type: string
+                  required:
+                  - name
+                  - podIP
+                  - state
                   type: object
-                volumeClaimTemplate:
-                  description: VolumeClaimTemplate defines the template to store WildFlyServer
-                    standalone data directory. The name of the template is derived
-                    from the WildFlyServer name.  The corresponding volume will be
-                    mounted in ReadWriteOnce access mode. This template should be
-                    used to specify specific Resources requirements in the template
-                    spec.
+                type: array
+                x-kubernetes-list-type: atomic
+              replicas:
+                description: Replicas is the actual number of replicas for the application
+                format: int32
+                type: integer
+              scalingdownPods:
+                description: "Represents the number of pods which are in scaledown
+                  process what particular pod is scaling down can be verified by PodStatus
+                  \n Read-only."
+                format: int32
+                type: integer
+            required:
+            - replicas
+            - scalingdownPods
+            type: object
+        type: object
+    served: true
+    storage: true
```

Comment:
that does not seem right: I'd expect that the new `v1alpha2` would be the storage version instead.

Reply:
You're right, a mistake, it is controlled by https://github.com/wildfly/wildfly-operator/pull/145/files#diff-553a69f3eee8fa0585c8bfe19a47a871R123